CMSIS/NN/Include/arm_nnfunctions.h

```diff
@@ -272,6 +330,56 @@ arm_status arm_convolve_s8(const cmsis_nn_context *ctx,
  */
 int32_t arm_convolve_s8_get_buffer_size(const cmsis_nn_dims *input_dims, const cmsis_nn_dims *filter_dims);
 
+/**
+ * @brief Basic s16 convolution function
+ * @param[in, out] ctx            Function context that contains the additional buffer if required by the function.
+                                  arm_convolve_s8_get_buffer_size will return the buffer_size if required
+ * @param[in]      conv_params    Convolution parameters (e.g. strides, dilations, pads,...).
+ *                                Range of conv_params->input_offset  : [-127, 128]
```

Comment:
could you double check that input/output zero points are applicable here. If not, please make a comment to that extent on the unused params.

Reply:
Done